modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= S3 - Compatible backup storage providers
+
+
+OADP supports a variety of object storage providers for different backup and snapshot operations.
+
+== S3 drivers for Backup Storage Locations
+
+=== Supported
+
+The following S3 compatible object storage providers, are fully supported by OADP through AWS plugin, for use as Backup Storage Locations:
+
```

Comment:
"AWs plugin" sounds like it is missing an article or should be pluralized, but I'm not sure. If it sounds right to you, keep it. In the worst case, the OCP doc team will suggest something different.

Reply:
Probably should be "the AWS plugin". It's a single plugin.